zer0-os/zOS
# Unite context panel app layout

### What does this do?
adds context so layout can be managed by platform

### Why are we making this change?
to allow the platform to manage layout and keep in sync, using direction from apps.

### How do I test this?
open the app. go to feed. open the context panel. go to channels. the context panel should stay open. if you close it, and go back to feed, it will stay closed.

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?
